addons/tidb/Chart.yaml

```diff
@@ -7,3 +7,8 @@ type: application
 version: 0.1.0-alpha.1
```

Comment:
The chart version should be changed to [tidb-0.8.0-alpha.5](https://github.com/apecloud/helm-charts/releases/tag/tidb-0.8.0-alpha.5)